app/assets/javascripts/components/course_creator/course_form.jsx

```diff
@@ -7,6 +7,36 @@ import CourseUtils from '../../utils/course_utils.js';
 import selectStyles from '../../styles/select';
 import WikiSelect from '../common/wiki_select.jsx';
 
+const createReactClass = require('create-react-class');
+
+const Application = createReactClass({
```

Comment:
This needs a better name, specific enough that seeing only the component name is enough that I know what it is for. It should probably be in a separate file as well.